codegenerator/cli/npm/envio/src/db/EntityHistory.res

```diff
@@ -335,7 +334,7 @@ let fromTable = (table: table, ~schema: S.t<'entity>): t<'entity> => {
     \${shouldCopyCurrentEntity});\``
 
   let insertFn: (Postgres.sql, Js.Json.t, ~shouldCopyCurrentEntity: bool) => promise<unit> =
-    insertFnString->Table.PostgresInterop.eval
+    insertFnString->eval
```

Comment:
@DZakh, I can't remember when this was added but don't you think eval should be at start up time and not at runtime of the insert function?